Source/WebCore/testing/Internals.cpp

```diff
@@ -4348,6 +4363,9 @@ bool Internals::isFromCurrentWorld(JSC::JSValue value) const
 JSC::JSValue Internals::evaluateInWorldIgnoringException(const String& name, const String& source)
 {
     auto* document = contextDocument();
+    if (!document || !document->frame())
+        return JSC::JSValue { };
```

Comment:
This is not right. We need to return something like `undefined` or `null` using a function like `jsUndefined()` or `jsNull()`, not `JSValue { }`.

Reply:
OK, it looks that I copied that from ScriptController::evaluateInWorldIgnoringException but indeed I meant to return null. Fixed.